Sources/TelemetryClient/Signal.swift

```diff
@@ -25,6 +25,7 @@ internal struct SignalPayload: Codable {
     var appVersion: String = Self.appVersion
     var buildNumber: String = Self.buildNumber
     var isSimulator: String = "\(Self.isSimulator)"
+    var isDebug: String = "\(Self.isDebug)"
```

Comment:
re-iterating from general PR comment: I'm not sure how necessary it is to add an isDebug key to the payload b/c presumably it will only be relevant for a very small number of signals actually sent to the server (and can be inferred from other payload key values). I am happy to remove that isDebug key addition if would make more sense.

Reply:
It doesn't hurt -- in fact I had an isDebug key in the payload for a long time and only recently removed it. It'll help people filter between debug and non debug signals